WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -13,42 +12,27 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 {
 	[AutoNotify] private string _amount;
 	[AutoNotify] private string _amountFiat;
-	[AutoNotify] private List<PrivacySuggestionBenefit> _benefits;
-	[AutoNotify] private PrivacyOptimisationLevel _optimisationLevel;
-	[AutoNotify] private bool _optimisationLevelGood;
+	[AutoNotify] private string _differenceFiat;
 
 	public ChangeAvoidanceSuggestionViewModel(decimal originalAmount,
 		BuildTransactionResult transactionResult,
-		PrivacyOptimisationLevel optimisationLevel,
 		decimal fiatExchangeRate,
-		bool isOriginal,
-		params PrivacySuggestionBenefit[] benefits) : base(transactionResult, isOriginal)
+		bool isOriginal) : base(transactionResult, isOriginal)
 	{
-		_optimisationLevel = optimisationLevel;
-		_benefits = benefits.ToList();
-
 		decimal total = transactionResult.CalculateDestinationAmount().ToDecimal(MoneyUnit.BTC);
 
 		_amountFiat = total.GenerateFiatText(fiatExchangeRate, "USD");
-		_optimisationLevelGood = optimisationLevel == PrivacyOptimisationLevel.Better;
 
-		if (_optimisationLevelGood)
+		if(!isOriginal)
 		{
 			var fiatTotal = total * fiatExchangeRate;
 			var fiatOriginal = originalAmount * fiatExchangeRate;
 			var fiatDifference = fiatTotal - fiatOriginal;
 
-			var difference = (fiatDifference > 0
+			_differenceFiat = (fiatDifference > 0
 					? $"{fiatDifference.GenerateFiatText("USD")} More"
```

Comment:
I think the ` More` text should be different color then amount for better readability.

Reply:
tried, doesnt look good, and dont want us to take a position on if sending more is good or bad.